web/components/issues/issue-detail/root.tsx

```diff
@@ -190,9 +193,9 @@ export const IssueDetailRoot: FC<TIssueDetailRoot> = observer((props) => {
       addIssueToCycle: async (workspaceSlug: string, projectId: string, cycleId: string, issueIds: string[]) => {
         try {
           await addIssueToCycle(workspaceSlug, projectId, cycleId, issueIds);
```

Comment:
adding and removing issue to cycle and module has their own spinner, can you try to remove that and use promise toast